core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -302,18 +523,16 @@ private int positionedReadInternal(long pos, byte[] b, int off, int len) throws
 
   @Override
   public long remaining() {
-    // WARNING: do not call remaining() directly within this file, use remainingInternal() instead
```

Comment:
is this warning no longer true?

Reply:
due to refactoring in `UnknownLengthFileInStream`, this warning is gone.